requirements/pip.txt

```diff
@@ -30,8 +30,8 @@ Pygments==2.9.0
 
 # Basic tools
 redis==3.5.3
-kombu==5.0.2
-celery==5.0.5
+kombu==5.1.0
+celery==5.1.0
```

Comment:
Lot of changes https://docs.celeryproject.org/en/v5.1.0/changelog.html, but nothing obvious that could break our code, we may want to skip the update in the next deploy (bc the current deploy has some refactors going live) or maybe just include it and test it locally.

Reply:
Yeap. I'm fine skipping this upgrade and make it in the following deploy after all the other upgrades.